pkg/services/alerting/engine.go

```diff
@@ -117,7 +117,10 @@ func (e *Engine) processJob(grafanaCtx context.Context, job *Job) error {
 			if err := recover(); err != nil {
 				e.log.Error("Alert Panic", "error", err, "stack", log.Stack(1))
 				ext.Error.Set(span, true)
-				span.LogFields(tlog.Error(fmt.Errorf("error: %v", err)))
+				span.LogFields(
+					tlog.Error(fmt.Errorf("%v", err)),
```

Comment:
fmt.Errorf is good if you need specific formatting, but in this case this is better:
```
tlog.Error(err) # if err is an error, which i think it is?
tlog.Error(errors.New(err)) # if err is a string
```

Reply:
https://golang.org/pkg/builtin/#recover

```recover()``` returns ```interface{}``` so we don't know the type.
Type assertion seems verbose while ```fmt.Errorf()``` will take care of it for me.